### DISCUSSION TITLE
Search for endpoints

### DISCUSSION THREAD

**Discussion by balfonzo** [mockoon/mockoon#372] [Ideas] [answered]
I have several endpoints that begging with the same paths, so it is hard to quickly find the endpoint that I want to modify. I am aware that I could on the notes tags or keywords, but i think it would be nice to have a search box to filter the endpoints based on the paths or the notes.

**Comment by 255kb** [comment, marked as answer]
Indeed, this would be great! I started working on a search/filter feature in #257 and have a local working POC. 
We are currently really busy with the CLI, but as soon as it's finished, the API search will be next!

**Reply by balfonzo** [reply]
Awesome! I'll keep an eye on the updates thanks.

**Reply by 255kb** [reply]
I'm only sorry that I cannot give an ETA. CLI is keeping us so busy! But we are getting there 🚀
